Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -37,12 +37,18 @@ static std::unordered_map<std::string, std::string> s_textureMap;
 static std::unordered_map<std::string, std::shared_ptr<HiresTexture>> s_textureCache;
 static std::mutex s_textureCacheMutex;
 static std::mutex s_textureCacheAquireMutex;  // for high priority access
-static Common::Flag s_textureCacheAbortLoading;
+static Common::Flag s_textureCacheAbortPrefetching;
 static bool s_check_native_format;
 static bool s_check_new_format;
 
 static std::thread s_prefetcher;
 
+static constexpr double CNV_BYT_TO_MB = (1024.0 * 1024.0);
+static constexpr double CNV_MS_TO_SEC = 1000.0;
+static constexpr double DOLPHIN_MIN_REQ_MEM = (1024.0 * 1024.0 * 1024.0 * 2.0);
+
+static u32 counter_textures_mips_amount;
```

Comment:
Any reason why this is a global? If it's just a load counter, it can be local to the load function,

Reply:
There are two voids involved. Otherwise there's some other way to do this, would need larger changes.

Reply:
Would just printing out s_textureMap.size() work for the overall count, then?

Reply:
I still have to figure out what all the "ptr", "level" "map" and more deeper things are and mean , I did made an attempt to kinda get an idea when I tried to replace some of the code that's used for loading textures, that lead me to a huge tree that I had to adapt which was way out of the scope of the cancelled PR, that was just a side research, so I'll have to do more of that to figure out more.